Dockerfiles/Dockerfile.init

```diff
@@ -3,15 +3,15 @@
 # some arguments that must be supplied
 ARG BASEIMAGE
 
+FROM golang:1.20 as builder
```

Comment:
AS

Reply:
The Dockerfile has this in lowercase as well. We can change in the future.